src/ipc.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { createIpc } from '@ipc';
+import type { BrowserWindow } from 'electron';
+
+import { REDUX_CHANNEL } from '@bridge';
+import { Process } from '@common/store';
+
+jest.unmock('@bridge');
+
+const mockWindow = { webContents: { send: jest.fn() } };
+const mockCrypto = { send: jest.fn(), on: jest.fn(), removeListener: jest.fn() };
+
+describe('createIpc', () => {
+  it('emits to browser window when targeting Renderer', () => {
+    const ipc = createIpc((mockWindow as unknown) as BrowserWindow, mockCrypto as any);
```

Comment:
Maybe use something like `mockCrypto as unknown as ReturnType<typeof createCryptoProcess>` to avoid `any`.